rilldata/rill
# fix: date picker locale and month bugs

## Checklist

## Summary
#### Issue addressed: 
Fixed bugs reported by @djbarnwal :
- `en-IN` was using wrong date format, causing "Invalid Date" to appear in picker
- prev/next month buttons were not working

#### Details:

## Steps to Verify
1. Open a dashboard
2. Use picker in different locale to confirm dates can be displayed correctly, selected, and applied
3. Use calendar month navigation buttons
